source/mapping-rules/mapping-rule-options/mapping-rule-filters.txt

```diff
@@ -0,0 +1,132 @@
+.. _rm-mapping-filters:
+
+====================
+Mapping Rule Filters
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In a migration project, you can use mapping rule javascript expressions to filter mapping rules based on document values. Mapping rule filters are available to all mapping rule types.
+
+Mapping rule options allow you to:
+
+- Filter mapping rules based on logical conditions.
+- Limit the number of documents written to a table.
+- Access complex and nested document structure for filtering.
+
+
+Examples
+-------------
+
+You can apply one filter per mapping rule. Each filter can have a javascript expression that when evaluated to ``true`` filters documents for migration.
+
+Below are examples of input documents and mapping rule javascript expressions that filter the data.
+
+
+Filter Based on Field Value
+—---------------------------------
+
+The following example filters a document based on a flat field value.
+
+Input:
+
+
+.. code-block:: none
```

Comment:
Formatting is messed up. Also it doesn't make sense to show the input as MongoDB documents. From the user's perspective these are all table rows. Can we show the raw data in a tabular form and then show an example where we create an array called `mexicanCustomers` that has the filter?